src/ImpurityDensities/ThermalDiffusionLithiumDensity.jl

```diff
@@ -31,24 +31,49 @@ struct ThermalDiffusionLithiumDensity{T <: SSDFloat} <: AbstractImpurityDensity{
     lithium_diffusivity_in_germanium::T
 end
 
-function calculate_lithium_diffusivity_in_germanium(lithium_annealing_temperature::T)::T where {T <: SSDFloat}
+struct LithiumDiffusionBranch{T <: SSDFloat}
+    T_min::T
+    T_max::T
+    D0::T
+    H::T
+end
+
+struct LithiumDiffusionParameters{T <: SSDFloat}
+    lowT::LithiumDiffusionBranch{T}
+    highT::LithiumDiffusionBranch{T}
+end
```

Comment:
This could then become a tuple with variable length `N`:

```suggestion
struct LithiumDiffusionParameters{T <: SSDFloat, N}
    parameters::NTuple{N, LithiumDiffusionBranch{T}}
end
```